webfscholar/web.py

```diff
@@ -17,6 +17,15 @@ def add_parser(subparsers, parent):
     parser_web.add_argument('--theme', choices=THEMES.keys(), default='montserrat-badges')
 
 
+def compare_year(pub):
+    if 'year' in pub:
+        return int(pub['year'])
+    else:
+        # If there's no year, it probably isn't very important, put it at the end.
+        print("WARNING: No publication date is available, this item may not sort properly.")
```

Comment:
I would've done `import logging` and `logging.error` so that user can control verbosity